src/schema/objects/enums.yaml

```diff
@@ -305,6 +306,13 @@ CTF:
 ElektaNeuromag:
   value: ElektaNeuromag
   display_name: Elekta Neuromag
+  description: |
+    RAS orientation and the origin between the ears.
+    [DEPRECATED](SPEC_ROOT/common-principles.md#definitions).
+    Dataset curators SHOULD use `NeuromagElektaMEGIN` instead.
```

Comment:
If we're going to deprecate, then we should have checks to warn curators to normalize. It seems this shows up in 7 metadata fields:

```console
$ git grep -B12 _MEGCoordSys | grep \ name:
src/schema/objects/metadata.yaml-  name: AnatomicalLandmarkCoordinateSystem
src/schema/objects/metadata.yaml-  name: DigitizedHeadPointsCoordinateSystem
src/schema/objects/metadata.yaml-  name: EEGCoordinateSystem
src/schema/objects/metadata.yaml-  name: FiducialsCoordinateSystem
src/schema/objects/metadata.yaml-  name: HeadCoilCoordinateSystem
src/schema/objects/metadata.yaml-  name: MEGCoordinateSystem
src/schema/objects/metadata.yaml-  name: NIRSCoordinateSystem
```

Could make a check in `src/schema/rules/checks/deprecations.yaml`:

```yaml
AnatomicalLandmarkCoordinateSystemDeprecation:
  issue:
    code: ELEKTA_NEUROMAG_DEPRECATED
    message: |
        "ElektaNeuromag" as a coordinate system is deprecated.
        Use "NeuromagElektaMEGIN" instead.
    level: warning
  selectors:
    - sidecar.AnatomicalLandmarkCoordinateSystem
  checks:
    - sidecar.AnatomicalLandmarkCoordinateSystem != "ElektaNeuromag"

DigitizedHeadPointsCoordinateSystemDeprecation:
  $ref: rules.checks.deprecations.AnatomicalLandmarkCoordinateSystemDeprecation
  selectors:
    - sidecar.DigitizedHeadPointsCoordinateSystem
  checks:
    - sidecar.DigitizedHeadPointsCoordinateSystem != "ElektaNeuromag"

...
```

Reply:
Thanks for the suggestions @effigies I agree and have created a PR for this, using your code. Please feel free to adjust if that's not what you had in mind. https://github.com/bids-standard/bids-specification/pull/1848